ambu_libraries/ambu_common/SensorDlcL20D4.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "SensorDlcL20D4.h"
+#include <Wire.h>
+#include <Arduino.h>
+
+#include <HardwareSerial.h>
+
+SensorDlcL20D4::SensorDlcL20D4 () : GenericSensor(DLC_L20D4_ADDR) { }
+
+void SensorDlcL20D4::setup() {
+
+   // Start new cycle for dlc
+   Wire.beginTransmission(addr_);
+   Wire.write(DLC_L20D4_CMD); // 2 cycle average = 8ms
+   Wire.endTransmission();
+}
+
+void SensorDlcL20D4::update(unsigned int ctime) {
```

Comment:
what is ctime for?

Reply:
I generally put the current time value as an arg to all of the update functions in case they need them. I think millis() is a relatively expensive call.